Project3_STL/src/Triangle.cpp

```diff
@@ -0,0 +1,29 @@
+#include "../headers/Triangle.h"
+using namespace std;
+namespace Shapes3D

```

Comment:
Same, check without namespace.

Reply:
I have updated cpp files as per your comment and it is working fine.